packages/ember/tailwind-webpack/index.js

```diff
@@ -0,0 +1,52 @@
+// @ts-check
+import { files, html, packageJson, project } from 'ember-apply';
+import path, { dirname } from 'path';
+import { fileURLToPath } from 'url';
+
+// @ts-ignore
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default async function run() {
+  await packageJson.addDevDependencies({
+    'tailwindcss': '^3.0.0',
+    'postcss': '^8.0.0',
+    'postcss-loader': '^7.0.0',
+    'autoprefixer': '^10.0.0'
+  });
+
+  // TODO: For some reason the css file in the [official guides](https://tailwindcss.com/docs/guides/emberjs) goes to `app/app.css` instead of `app/styles/app.css`. Do we want to honour that?
+  // TODO: I think we should move the `postcss.config.js and tailwind.config.js to app/config, but how to make postcss/tailwind then find them?
+  await files.applyFolder(path.join(__dirname, 'files'), '.');
+
+  await html.insertText('ember-cli-build.js', {
```

Comment:
`html`? does this work :sweat_smile: 

it was intended that this would work this way, but I'm impressed it it does!

the way I've lately been doing easy code insertions is with something like:
https://astexplorer.net/#/gist/3c9f270caee5835a7f314592a9f8b3bb/46dd245b3bd642a1e92002808872b7bebfff37b6
(`j.template.expression`)